www/heatmap-demo.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        html, body, {
+            position: fixed;
+            top: 0;
+            bottom: 0;
+            left: 0;
+            right: 0;
+        }
+
+        .grid-container {
+            display: grid;
+            grid-template-columns: auto auto;
+            grid-gap: 10px;
+            background-color: #ffffff;
+            padding: 0px;
+        }
+
+        .grid-container > div {
+            background-color: rgba(255, 255, 255, 1);
+            text-align: center;
+            padding: 2px 0;
+            font-size: 30px;
+        }
+    </style>
+
+</head>
+<body>
+    
+    <div id="heatmapContainer" style="margin: auto; width: 100%; height: auto; max-width: 700px">
+            <img id="webgazer-pic" src="./media/heatmap-demo.png" style="width: 100%; height: auto; max-width: 700px">
+    </div>
+
+    <script src="./js/heatmap.js"></script>
+    <script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
```

Comment:
Don't we require jQuery in the package.json? We should use it. Otherwise it could cause unnecessary CORS issues if someones tries to deploy the webpage on the server. At the very least we should be consistent with the jQuery version we are using.

Reply:
I think we're also calling jQuery using URLs in some of our other HTML files? I'll go and change them too so everything is consistent.

Reply:
References to JQuery have been updated in the other files (index.html & calibration.html) on master, you can use them as guidance for this.

Reply:
This still hasn't been changed @elviskahoro